installer/inventory

```diff
@@ -119,6 +119,11 @@ create_preload_data=True
 # your credentials
 secret_key=awxsecret
 
+# By default a broadcast websocket secret will be generated.
```

Comment:
Is @chrismeyersfsu is this a _problem_, per-se?

Reply:
Only problem I can think that this may cause is if the `uwsgi` service isn't restarted. I could imagine the `uwsgi` workers might restart periodically and some pick up the new password while some use the old password.

Other than that, no. The user shouldn't have to manage or care about the secret being generated and exchanged.